### ISSUE TITLE
Remove "rubypick"

### ISSUE THREAD

**Issue by voxik** [minimization/content-resolver-input#343]
rubypick is on the [unwanted](https://github.com/minimization/content-resolver-input/blob/master/configs/sst_cs_apps-unwanted-ruby.yaml#L13) list. It should be easy to get rid of it, because there is feature flag in the [.spec](https://src.fedoraproject.org/rpms/ruby/blob/master/f/ruby.spec#_71) file. The only question is where to put the conditional. Is there possible to set them in Koji tag or is there some macro file with the feature flags? @tdawson

**Comment by voxik** [comment]
Actually what I don't understand, why ELN expands the `%{?fedora}`. I think this is wrong and ELN should not do this. The flag is there specifically for Fedora, which has richer ecosystem and therefor it might benefit from rubypick.

**Comment by tdawson** [comment]
I'm confused about this issue.
I don't see rubypick in the content-resolver.  Where are you seeing it?
Also, ELN does NOT expand %{?fedora}.  Originally we had set it to 0 or possibly %null, but that was causing problems, so we completely remove it.  Again, where are you seeing this so if there is a bug we can fix it.

**Comment by tdawson** [comment]
Ahh ... I think I found it.  It's in the list of packages we are originally building for RHEL9.
That is an old list ... depending on which one they are using it might have been before we fixed a bug in the buildroot generator.
When discussed about building some of the packages we know we don't want (there are more than just rubypick) we were told to just go ahead and build that list and we will test the process for removing them.
